spring/spring6/spring6-common/src/main/java/org/infinispan/spring/common/config/SpringCacheMode.java

```diff
@@ -0,0 +1,7 @@
+package org.infinispan.spring.common.config;
+
+public enum SpringCacheMode {
```

Comment:
not sure this is useful @tristantarrant I did not use it 

I want to do this for the spring boot starter, not sure is useful for bare spring apps

Reply:
maybe is not hard to do it for bare spring apps if we use the properties as you did for read and write async. same logic. but I don't want to test if "reactor" is there (auto mode). I rather tell: if you need reactive, put this property.